<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCC_Recorder</class>
 <widget class="QMainWindow" name="MCC_Recorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ScanDevButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan
Devices</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Device_dropdown">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="Analog_plot" name="Channel_viewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>1101</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Analog_plot</class>
   <extends>QWidget</extends>
   <header>plotting</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
